features/step_definitions/asset_host_steps.rb

```diff
@@ -1,25 +1,42 @@
 # frozen_string_literal: true
 
+Given /^the asset host is unset$/ do
+  Rails.configuration.three_scale.asset_host = nil
+end
+
 Given /^the asset host is set to "(.*)"$/ do |asset_host|
   cdn_url = "#{asset_host}:#{Capybara.current_session.server.port}"
-  Rails.configuration.asset_host = cdn_url
   Rails.configuration.three_scale.asset_host = cdn_url
 end
 
 Then /^((javascript|font)\s)?assets should be loaded from the asset host$/ do |asset_type|
-  cdn_url = Rails.configuration.asset_host
+  cdn_url = Rails.configuration.three_scale.asset_host
```

Comment:
maybe assert here that the value is `#present?`?

Reply:
Done